db/pre-re/status.yml

```diff
@@ -137,6 +144,12 @@ Body:
       Gvg_Stun: true
     End:
       Dancing: true
+      Stone: true
```

Comment:
Any OPT1 status change should end any other OPT1 status change (unless it makes it fail). Do we still want to define "End" here?